baseballAPI.UnitTests/Controllers/PlayerControllerTests.cs

```diff
@@ -25,18 +24,23 @@ public void SetUp()
         }
 
         [Test]
-        public void ReturnsPlayerID()
+        public void ReturnsPlayerId()
```

Comment:
might want to update this to reflect that you are returning full `People` objects.